src/api.ts

```diff
@@ -1,13 +1,15 @@
 import express from 'express';
 import { capture } from '@snapshot-labs/snapshot-sentry';
-import { getSpace } from './helpers/spaces';
+import refreshSpacesCache, { getSpace } from './helpers/spaces';
 import { name, version } from '../package.json';
 import { sendError } from './helpers/utils';
 
 const router = express.Router();
 const network = process.env.NETWORK || 'testnet';
 const SEQUENCER_URL = process.env.SEQUENCER_URL || '';
 
+refreshSpacesCache();
```

Comment:
Why this one here and `refreshModerationData` in index.ts 😅 better to keep all in one place